homework.md

```diff
@@ -44,70 +44,73 @@ Using CSS, how would I select every `<h1>` tag on the page?
 
 What is JavaScript?
 
-A programming language used on the internet
+An object-oriented scripting language commonly used to create interactive effects within web browsers.
```

Comment:
Hey Dharma, as far as I know JavaScript is a functional programming language. JavaScript is not a class-based object-oriented language. But it still has ways of using object oriented programming